content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -269,7 +281,95 @@ This returns the same output as checking the flow status on Swagger UI. You see:
 
 If you have any questions about the Corda Node CLI commands, run the `--help` command for more information.
 
+## Set up and test the UI
+
+The Solar System CorDapp comes with a built-in UI. See the <a href="https://github.com/corda/samples-kotlin-corda5/tree/main/Tutorial/solarsystem/Web-UI">`Web-UI`</a> folder of the sample folder to study the code.
+
+### Before you start
+
+Before you can build the Solar System CorDapp UI, you must:
+
+* [Deploy your CorDapp to a local Corda 5 network](*deploy-the-cordapp-using-corda-cli)
+* Download the [Node.js](https://nodejs.org/en/download/) asynchronous event-driven JavaScript runtime for your platform.
+* Ensure that port `3000` is not being used by any other applications. The UI is hard-coded to run on port `3000`.
```

Comment:
I forgot to mention that it's the proxy that is hardcoded to listen on port 3000, and the reason there's a proxy is due to the fact that cors header won't be available from the server till dev preview 2, this cors information has been put in the docs somewhere else so not sure if it would need to be mentioned here also or it's ok that it's not?

Reply:
Ooh this is good to know. Checking to see if we've included this in other customer-facing docs. Thanks, @emisrz !